lib/nostrum/consumer.ex

```diff
@@ -102,6 +102,7 @@ defmodule Nostrum.Consumer do
   @type message_update :: {:MESSAGE_UPDATE, {updated_message :: Nostrum.Struct.Message.t}, ws_state}
   @type message_reaction_add :: {:MESSAGE_REACTION_ADD, map}
   @type message_reaction_remove :: {:MESSAGE_REACTION_REMOVE, map}
+  @type message_reaction_remove_all :: {:MESSAGE_REACTION_REMOVE_ALL, map}
```

Comment:
This, and all of the other types here should be in the format of {event_type , {things_to_dispatch}, ws_state}.

I see some of the surrounding types are still incorrect here, could you fix them up to more in line with the rest? Thanks!

Reply:
Ah alright. I based it off of the previous ones about reactions. I'll fix em up in my next commit

Reply:
No problem! I'm the one that lead you astray. 😅